### DISCUSSION TITLE
donate/

### DISCUSSION THREAD

**Discussion by giscus** [rime/home#1220] [General]
# donate/

Le blog de la Rime

https://rime.im/donate/

**Comment by JagoWang** [comment]
支持！

**Comment by ming1007520388** [comment]
输入法非常棒,打字的流畅度和自动造词等功能真的很方便. 经过一年多的使用 我已完全转向了ubuntu.再次感谢您.

**Comment by Patricivs** [comment]
㨔㧢

**Comment by jeidyknight** [comment]
终于找到能比较好适配rime的自然码双拼码表和词库了，赶紧来这里表示一下

**Comment by Elflare** [comment]
太好用了，小捐一点以示心意

**Comment by houchen-li** [comment]
从16年开始使用 Arch Linux ，那时接触了中州韵，这个输入法不像搜狗那些国内全家桶：那类输入法总想着往用户电脑里塞一大堆自己的编程框架私货，臃肿不堪。中州韵充分利用了 Linux 已有图形环境和 fcitx 输入法框架，实现了非常丰富的特性，这么多年一直在我的开发环境默默稳定运行。

今天路过开发团队论坛，特赞助100元！希望 Rime 中州韵能够坚持自己的道路，光大我汉家文字！

**Reply by lotem** [reply]
![gongdexiang](https://github.com/user-attachments/assets/21cf435b-c053-408d-8bc8-6d2b32610d66)

**Comment by modan1998** [comment]
支持

**Comment by TBRay90** [comment]
太棒了，必须支持！

**Comment by weic6060** [comment]
![微信圖片_20260119102451_190_7](https://github.com/user-attachments/assets/163ec4f6-aa7e-4bda-a118-2e90f73df7e8)
不小心用上了，幫只會拼音的同事能在中文ERP中正確輸入中文繁體字，附上小小的心恴。

**Reply by lotem** [reply]
![](https://private-user-images.githubusercontent.com/151157/447875364-21cf435b-c053-408d-8bc8-6d2b32610d66.jpg?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Njg4NzE0MzAsIm5iZiI6MTc2ODg3MTEzMCwicGF0aCI6Ii8xNTExNTcvNDQ3ODc1MzY0LTIxY2Y0MzViLWMwNTMtNDA4ZC04YmM4LTZkMmIzMjYxMGQ2Ni5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjYwMTIwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI2MDEyMFQwMTA1MzBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jMTk5ZTJkM2UwZDIzYzI3MzMzNWNkZjc3YTJmZGZhM2Y0NGU1ODY0ODkxNzAyZjk5Y2Q2ZTNmYTE4ZmQ3ZGVmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.KEmgusAKwaKJbGoaXGgRl9x9iePErjjDwKtdtIDH14o)

**Comment by skirtdress** [comment]
好 彩

**Comment by yizhiru** [comment]
输入法简洁好用，支持一下！

**Comment by DecarbonizedGlucose** [comment]
好

**Comment by kjstart** [comment]
捐过啦, 谢谢啊, 很好用:D
